iceberg/iceberg-handler/src/test/results/positive/row_count.q.out

```diff
@@ -0,0 +1,302 @@
+PREHOOK: query: drop table llap_orders
```

Comment:
Please mask out the values that can change frequently, like `current-snapshot-id`, `added-files-size`, etc...

Reply:
Fixed